__tests__/app.test.js

```diff
@@ -6,161 +6,405 @@ const db = require('../db/connection');
+      .post('/api/articles/5/comments')
+      .send({ name: '', body: 'test' })
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe('Bad Request');
+      });
+  });
+  test('returns 400 bad request if body key is empty', () => {
+    return request(app)
+      .post('/api/articles/5/comments')
+      .send({ name: 'butter_bridge', body: '' })
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe('Bad Request');
+      });
+  });
+  test('returns 400 bad request if values are the wrong data type', () => {
+    return request(app)
+      .post('/api/articles/5/comments')
+      .send({ name: 'butter_bridge', body: 5 })
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe('Bad Request');
+      });
+  });
+});
 
-    describe('GET /api/articles/:article_id/comments', () => {
-        test('Returns an array of comment objects for the given article_id, containing comment_id, votes, created_at, author (username from users table, body', () => {
-            return request(app)
-            .get('/api/articles/3/comments')
-            .expect(200)
-            .then((response) => {
-                expect(response.body.comments.length).toBe(2);
-                response.body.comments.forEach(comment => {
-                    expect(comment).toMatchObject({
-                        comment_id: expect.any(Number),
-                        votes: expect.any(Number),
-                        created_at: expect.any(String),
-                        author: expect.any(String),
-                        body: expect.any(String),
-                    })
-                })
-            })
-        })
-        test('Returns 404 not found when given an article id that does not exist', () => {
-            return request(app)
-            .get('/api/articles/222/comments')
-            .expect(404)
-            .then((response) => {
-                expect(response.body.msg).toBe('Not Found');
-            })
-        })
-        test('Returns 200 and empty body when selecting an article with no comments', ()=> {
-            return request(app)
-            .get('/api/articles/2/comments')
-            .expect(200)
-            .then((response) => {
-                expect(response.body.comments).toEqual([]);
-            })
-        })
-        test('Returns 400 bad request if a non integer is given as article_id', () => {
-            return request(app)
-            .get('/api/articles/a/comments')
-            .expect(400)
-            .then((response) => {
-                expect(response.body.msg).toBe('Bad Request');
-            })
-        })
-    })
+describe('DELETE /api/comments/:comment_id', () => {
+  test('Deletes a comment of the given id', async () => {
+    return request(app)
+      .del('/api/comments/10')
+      .expect(204)
+      .then((response) => {
+        return db.query(`SELECT * FROM comments;`).then((testRes) => {
```

Comment:
Really like this test that makes sure it's been deleted. 😍 
(You can get rid of the `async` if you're not using it just to be picky though)